### ISSUE TITLE
How do I find the correct registry and image name to use?

### ISSUE THREAD

**Issue by Simbaclaws** [mr-smithers-excellent/docker-build-push#187]
Hi,

I'm looking at the documentation and I'm trying to figure out how to use my private repository url with docker hub...

My private repository url looks like: account-name/repository-name
Instead of: repository-name/image-name

What should I use in this case?

Shouldn't the image be: account-name/repository-name/image-name then?

I am currently getting the following error message in my github action:
Error response from daemon: Get "https://registry-1.docker.io/v2/": unauthorized: incorrect username or password

I am using the:
docker.io registry, if that is correct?

even though I can login to dockerhub with that username and password...

**Comment by Simbaclaws** [comment]
Ok so I figured I did something wrong with my repository/image names.

I got that part solved... However, it looks like the directory flag simply adds the directory to the end of the command like so:
```
docker build -f Dockerfile -t docker.io/***/staging-backend:"v1.0.11" -t docker.io/***/staging-backend:latest .
```
I have a repository where the directories are:
- frontend
- backend

And I have a Dockerfile in those directories.

However, this build command seems to fail.

I have gone with a different github actions package instead where I'm using the context to solve the issue.
